### ISSUE TITLE
Finalize the DB

### ISSUE THREAD

**Issue by NickStafford2** [csims17/MovieDB#2]
By the end of the day I would like the DB layout and seeding to be finalized. If we want to make any changes, we have to make them now because changing them later will cause problems.

**Comment by cmehring** [comment]
I agree, getting everything so that the next two days will be easy will put
us ahead.
